haas/model.py

```diff
@@ -227,6 +182,31 @@ def get_console(self):
     def get_console_log_filename(self):
         return '/var/run/haas_console_logs/%s.log' % self.ipmi_host
 
+class Obm(AnonModel):
+    """ Obm superclass supporting various drivers 
+        related to out of band management of servers """
+    type = Column(String, nullable=False)
+
+    __mapper_args__ = {
+            'polymorphic_identity': 'new_obmType',
```

Comment:
You don't actually need `'polymorphic_identity'` here, because you're never going to be instantiating `Obm` itself, just subclasses. Please remove it.